src/ChirpInfrastructure/CheepRepository.cs

```diff
@@ -24,18 +24,22 @@ public class CheepRepository(ChirpDBContext context) : ICheepRepository
 
   }
   Above will be relevant later*/
-  public List<CheepDTO> ReadCheeps()
+  public List<CheepDTO> ReadCheeps(int pageNumber)
   {
-	var cheepList = new List<CheepDTO>();
-
+	//var cheepList = new List<CheepDTO>();
```

Comment:
Do we need to remove this in the future?

Reply:
Wasn't certain. Don't believe so.